src/libs/elektra/kdb.c

```diff
@@ -693,6 +726,15 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 
 	ksRewind (ks);
 
+	if (elektraGetDoUpdate2 (split, ks, parentKey) == -1)
```

Comment:
elektraSplitMerge needs to be called before spec, and elektraSplitGet before elektraSplitMerge. So i guess i'd have to adjust elektraSplitGet but i'm not sure how to do the part with the syncbits